openshift/e2e-tests-openshift.sh

```diff
@@ -17,7 +17,7 @@ if [[ "${PULL_BASE_REF:-}" == "release-next" ]]; then
   (( !failed )) && install_serverless_operator_release_next || failed=1
   (( !failed )) && run_client_e2e_tests serving || failed=1
   # Eventing setup & tests
-  (( !failed )) && install_knative_eventing_branch "${EVENTING_BRANCH}" || failed=1
+  # (( !failed )) && install_knative_eventing_branch "${EVENTING_BRANCH}" || failed=1
```

Comment:
You need to keep it for now?

Reply:
Not, really. I'll do more clean ups on the script afterwards.

Reply:
Fair point here :)